tests/controllers/UI/CalendarsControllerTest.php

```diff
@@ -30,6 +30,32 @@ public function testFailOnDestroyWithWrongId()
     }
 
     /**
+     * @test
+     */
+    public function testPriorityChange() {
```

Comment:
Make name of test explaining what the test is doing (length is of no matter)
for example: testPriorityOfCalendarChangesWhenASiblingIsRemoved()